### PR TITLE
Improved to operate whenever the spinbox arrow button is pressed

### PR DESCRIPTION
**Pull request by gdinolee**
Improved to operate immediately whenever the spinbox arrow button is pressed.

https://github.com/Samsung/ioter/issues/100

### CODE REVIEW

File: src/config.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "option_menu_shown": false,
+    "option_menu_shown": true,
```
Comment (junyong-sim) [inline src/config.json]: Should be reverted

Comment (junyong-sim) [reply src/config.json]: I've checked another commit that included the fix reverting this line.

File: src/things/humidity.py [hunk 2]
```diff
@@ -82,20 +82,24 @@ def post_setup_window(self):
...
         self.doubleSpinBoxInput = common_window.doubleSpinBoxInput
         self.textBrowserLog = common_window.textBrowserLog
         self.labelStatePicture = common_window.labelDevicePicture
 
     ## Initialise UI Input Button ##
     def init_spinbox(self):
         self.doubleSpinBoxInput.installEventFilter(self)
+        self.doubleSpinBoxInput.valueChanged.connect(self.spin_value_changed)
+        self.doubleSpinBoxInput.setSingleStep(1)
+        # self.doubleSpinBoxInput.setRange(0, 100)
```
Comment (junyong-sim) [inline src/things/humidity.py:94]: Doesn't `input box` need to set range ? If so, how about removing this comment ?

File: src/config.json (hunk 1 above)
Comment (gdinolee) [reply src/config.json]: Thanks alot

File: src/things/humidity.py (hunk 2 above)
Comment (gdinolee) [reply src/things/humidity.py]: I fixed it . appreciate it

**Review by junyong-sim — APPROVED** [review]
LGTM

File: src/things/lightsensor.py [hunk 3]
```diff
@@ -158,7 +165,7 @@ def update_ui(self):
     def send_command(self):
         LightsensorCommand.set_lightsensor(
             self.device_info.device_num, self.measured_value)
-        self.get_window().appendTextBrowserLog(f'[Send] {self.spinBoxInput.value()}{LIGHTSENSOR_UNIT}')
+        self.textBrowserLog.append(f'[Send] {self.spinBoxInput.value()}{LIGHTSENSOR_UNIT}')
```
[the remark this answers is not in the document]
Comment (gdinolee) [reply src/things/lightsensor.py]: I fixed it

File: src/things/lightsensor.py [hunk 4]
```diff
@@ -92,19 +92,20 @@ def get_ui_component_from_common_window(self, common_window):
...
+        self.horizontalSliderLightsensor.setRange(MEASURED_VALUE_MIN, MEASURED_VALUE_MAX)
+        self.horizontalSliderLightsensor.setSingleStep(1000)
+        self.horizontalSliderLightsensor.setPageStep(1000)
         self.horizontalSliderLightsensor.setValue(self.measured_value)
         self.horizontalSliderLightsensor.sliderPressed.connect(
             self.sliderPressed)
         self.horizontalSliderLightsensor.sliderReleased.connect(
             self.sliderReleased)
-        self.horizontalSliderLightsensor.valueChanged.connect(
+        self.horizontalSliderLightsensor.valueChanged.connect(ㅗ
```
[the remark this answers is not in the document]
Comment (gdinolee) [reply src/things/lightsensor.py]: I fixed it

**Pull request merged**